bids/tests/test_visualize.py

```diff
@@ -0,0 +1,153 @@
+from pathlib import Path
+
+import shutil
+
+import pytest
+
+from bids.layout import BIDSLayout
+from bids.visualize import EventPlotter  # type: ignore
+from bids.visualize import LayoutPlotter  # type: ignore
+from bids.tests import get_test_data_path
+
+"""TODO
+-   test on datasets with a large range of events types
+"""
+
+
+@pytest.mark.parametrize(
+    "output_dir",
+    [
+        (Path(__file__).parent.joinpath("tmp")),
+        (None),
+    ],
+)
+@pytest.mark.parametrize(
+    "show",
+    [
+        (True),
```

Comment:
What does `show=True` do during tests? Do we need to be mocking `matplotlib.figure.Figure.show()` to ensure that it is called but doesn't actually attempt to display things?

Reply:
I was using the test to visualize the figures directly.
added a global variable to toggle that rather than a pytest parametrization.